packages/terra-date-picker/tests/wdio/date-picker-spec.js

```diff
@@ -484,6 +515,205 @@ Terra.describeViewports('Date Picker', ['medium'], () => {
     Terra.it.validatesElement();
   });
 
+  describe('Initial Focus', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-date-picker/date-picker/date-picker-default');
+      browser.setValue('input[name="terra-date-month-date-input"]', '12');
+      browser.setValue('input[name="terra-date-day-date-input"]', '18');
+      browser.setValue('input[name="terra-date-year-date-input"]', '1994');
+      Terra.hideInputCaret('input[name="terra-date-month-date-input"]');
+      Terra.hideInputCaret('input[name="terra-date-day-date-input"]');
+      Terra.hideInputCaret('input[name="terra-date-year-date-input"]');
+    });
+
+    it('Month input focused', () => {
+      browser.click('input[name="terra-date-month-date-input"]');
+    });
+
+    Terra.it.validatesElement('month input highlighted');
+
+    it('Month input cleared with single Delete key press and loses highlight', () => {
+      browser.keys('Delete');
+    });
+
+    Terra.it.validatesElement('month input cleared completely and loses highlight');
+
+    it('Day input focused', () => {
+      browser.click('input[name="terra-date-day-date-input"]');
+    });
+
+    Terra.it.validatesElement('day input highlighted');
```

Comment:
`Terra.it.validatesElement` is deprecated. It is replaced by `Terra.validates.element` which must be wrapped inside an `it` block.

Reply:
Updated here : https://github.com/cerner/terra-framework/pull/1208/commits/a3ba740adba9699efbe60c5cfee0e655ce7f2489